x-pack/plugins/saved_objects_tagging/public/management/components/table.tsx

```diff
@@ -0,0 +1,193 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import React, { useRef, useEffect, FC } from 'react';
+import { EuiInMemoryTable, EuiLink } from '@elastic/eui';
+import { Action as EuiTableAction } from '@elastic/eui/src/components/basic_table/action_types';
+import { i18n } from '@kbn/i18n';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { TagsCapabilities, TagWithRelations } from '../../../common';
+import { TagBadge } from '../../components';
+
+interface TagTableProps {
+  loading: boolean;
+  capabilities: TagsCapabilities;
+  tags: TagWithRelations[];
+  selectedTags: TagWithRelations[];
+  onSelectionChange: (selection: TagWithRelations[]) => void;
+  onEdit: (tag: TagWithRelations) => void;
+  onDelete: (tag: TagWithRelations) => void;
+  getTagRelationUrl: (tag: TagWithRelations) => string;
+  onShowRelations: (tag: TagWithRelations) => void;
+}
+
+const tablePagination = {
+  initialPageSize: 20,
+  pageSizeOptions: [5, 10, 20, 50],
+};
+
+const sorting = {
+  sort: {
+    field: 'name',
+    direction: 'asc' as const,
+  },
+};
+
+export const isModifiedOrPrevented = (event: React.MouseEvent) =>
+  event.metaKey || event.altKey || event.ctrlKey || event.shiftKey || event.defaultPrevented;
```

Comment:
I think we have this in 3 or 4 spots around Kibana now... Should we extract it into something reusable?

Reply:
Could move to kibana_react. However most of the current usages are in core code, meaning that it couldn't import the util from the kibana_react bundle. And core is framework agnostic, so we can't expose that as an helper from core bundle or API.

Ideally it would be centralized, I agree, but it's only 3 lines, we probably have biggest factorization / centralization concerns before addressing this one.